src/server/templates/end.njk

```diff
@@ -2,7 +2,20 @@
 
 {% block body_main_content %}
   <div class="page-section">
-    <h1>Thank you</h1>
-    <p>Goodbye</p>
+    <h1 class="heading-medium">Thank you for applying to register</h1>
+
+    <div class="reading-width">
+      <p>They are dealing with your registration application. They will contact
+        you about it when they have completed processing it.</p>
+
+      <div class="callout callout--attention">
+        <p><strong>If you need to see a doctor sooner, contact the practice or <a href="http://www.nhs.uk/service-search">find
```

Comment:
"contact the practice on {{ practiceTelephone }}" ? Again, if it's a different batch of work, fine.